Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,28 @@
 import React from 'react';
 import './Header.css';
+
+import ImageThumbnail from './ImageThumbnail';
+import HeaderTitle from './HeaderTitle';
+import HeaderContent from './HeaderContent'
+import CardContainer from '../CardComponents/CardContainer'
+
+const HeaderContainer = () => {
+	return (
+
+		<header className="headerContainer"> 
+
+			<ImageThumbnail />
+			<div className="body">
```

Comment:
i would caution against using reserved html keywords like "body" for css classnames, my suggestion is to go more semantic like `header-body` or something similar